light-model-client/src/commonMain/kotlin/org/modelix/client/light/LightModelClient.kt

```diff
@@ -1,14 +1,21 @@
 package org.modelix.client.light
 
+import io.ktor.client.*
+import io.ktor.client.plugins.websocket.*
+import kotlinx.coroutines.delay
 import org.modelix.model.api.*
 import org.modelix.model.area.IArea
 import org.modelix.model.area.IAreaListener
 import org.modelix.model.area.IAreaReference
 import org.modelix.model.server.api.*
+import kotlin.time.Duration.Companion.milliseconds
+import kotlin.time.Duration.Companion.seconds
 
 private const val TEMP_ID_PREFIX = "tmp-"
 
-class LightModelClient(val connection: IConnection, val debugName: String = "") {
+class LightModelClient(
+    val connection: IConnection = WebsocketConnection(HttpClient { install(WebSockets) }, "ws://localhost:48302/ws"),
```

Comment:
We should consider that someone want to change host and port for the default websocket connection.